From 9d42640d413262d685211bb099bbbb6e58239523 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Thu, 9 Jul 2020 19:39:12 +0200
Subject: [PATCH] keytable: Fix missing inclusion of argp.h

error_t is only defined by argp.h on non-glibc systems.

Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>
---
 utils/keytable/bpf_load.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/keytable/bpf_load.c b/utils/keytable/bpf_load.c
index 9f64cf4e91e8..7ae9af4a57be 100644
--- a/utils/keytable/bpf_load.c
+++ b/utils/keytable/bpf_load.c
@@ -13,6 +13,7 @@
 #include <stdlib.h>
 #include <linux/bpf.h>
 #include <assert.h>
+#include <argp.h>
 #include "keymap.h"
 #include "bpf.h"
 #include "bpf_load.h"
-- 
2.27.0

